src/libs/actions/Modal.ts

```diff
@@ -85,4 +85,8 @@ function willAlertModalBecomeVisible(isVisible: boolean, isPopover = false) {
     Onyx.merge(ONYXKEYS.MODAL, {willAlertModalBecomeVisible: isVisible, isPopover});
 }
 
-export {setCloseModal, close, onModalDidClose, setModalVisibility, willAlertModalBecomeVisible, setDisableDismissOnEscape, closeTop};
+function getModalVisibleCount() {
```

Comment:
I propose we call this something like `allModalsHidden` and make it return closeModals.length === 0, since we don't care about the number of visible modals, all we care about is whether there's at least one modal visible

Also please add a comment explaining what this function does

Reply:
Good idea, but I prefer to not give it a comment since the name itself is already self-explanatory.